.circleci/config.yml

```diff
@@ -55,9 +55,13 @@ jobs:
       - run: |
           docker build -t envirodgi/processing:$CIRCLE_SHA1 .
           docker build -t envirodgi/processing:latest .
+          docker build -t envirodgi/ia-healthcheck:latest -f Dockerfile.ia-healthcheck .
+          docker build -t envirodgi/ia-archiver:latest -f Dockerfile.ia-archiver .
```

Comment:
Can you also re-tag these as `envirodgi/xyz:$CIRCLE_SHA1`? Not having unique tags makes rollbacks, reproduceability, etc. really hard. I’d go so far as saying the hash-based tag is more important than `latest` if we’re only going to have one.

Reply:
Yep, good catch, no problem.

Reply:
Actually, looking through this seems like the only thing we can do is what's already being done with envirodgi/processing, and building two images. One for archiving as an artifact and one labeled "latest", and the latest tag is what's actually pulled in  by web-monitoring-ops.

https://github.com/ibuys/web-monitoring-ops/blob/5f0efef84d0419185e6aabb86f59942b19d15db1/templates/staging/ia-archiver-cronjob.yaml#L13

Am I missing something, or does that sound good to you?

Reply:
Right, part of what I was getting at here was that ops *should not use `:latest`.* But that’s a matter for edgi-govdata-archiving/web-monitoring-ops#22, and it looks like you’ve done what needs doing here :)